### PR TITLE
Add  SenseCAP M1 to Hotspots App

### PR DESCRIPTION
**Pull request by autume**
Related : https://github.com/helium/HIP/pull/179

### CODE REVIEW

**Review by autume — COMMENTED** [review]
add SenseCAP M1 hotspot

**Review by autume — COMMENTED** [review]
Add SenseCAP M1 hotspot

**Review by autume — COMMENTED** [review]
modify picture and text for SenseCAP M1

**Review by autume — COMMENTED** [review]
Merge branch 'main' into sensecap-branding/m1

File: src/locales/en.ts [hunk 1]
```diff
@@ -195,6 +196,8 @@ export default {
...
       longap_p_1:
         '<b><white>Diagnostic support allows LongAP to identify issues with your Hotspot in a secure way.</white></b>\n\nLongAP will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@longap.com</b></purple> from the email used to purchase the Hotspot.',
+      seeed_p_1:
+        '<b><white>Diagnostic support allows us to identify issues with your M1 gateway in a secure way.</white></b>\n\nWe will never access to private keys and will only ever be able to access your M1 gateway and not other devices on your network.\n\nIf you would like to opt-out of diagnostic support, please email  <purple><b>SenseCAP@seeed.cc</b></purple>  and provide us with your purchase infomation and detailed problem discription to get support. ',
```
Comment (cokes518) [inline src/locales/en.ts:200]: please double check typos, a few spelling mistakes here.

File: src/locales/en.ts [hunk 2]
```diff
@@ -226,6 +231,8 @@ export default {
         'The SyncroB.it Hotspot LED bar will light up blue once it’s powered on.',
       longapone_subtitle_2:
         'The LongAP One PWR LED will light up once it’s powered on.',
+      seeed_subtitle_2:
+        'About 30 seconds, the blue indicator light on the back of M1 changes from off to fast flash mode, indicating that the M1 gateway is ready.',
```
Comment (cokes518) [inline src/locales/en.ts:235]: please change to "In 30 seconds, the blue indicator.."

**Pull request closed without merging**